conformance/conformance.proto

```diff
@@ -83,7 +91,10 @@ message ConformanceRequest {
   // protobuf_test_messages.proto2.TestAllTypesProto2.
   string message_type = 4;
 
-  bool ignore_unknown_json = 5;
+  // Each test is given a unique test category. Some category may need spedific
```

Comment:
"Each test is given a unique test category" seems to indicate each test has a different category...

Reply:
"spedific"?